src/fairseq2/metrics/recorders/jsonl.py

```diff
@@ -52,6 +52,11 @@ def __init__(
     def record_metric_values(
         self, category: str, values: Mapping[str, object], step_nr: int | None = None
     ) -> None:
+        """
+        Gets, sorts, and maps metrics, their values, and descriptions to a ``dict``
```

Comment:
Again, no need to override the docstr of base interface. Any behavior specific to this subclass should ideally be described in the class docstr.